### PR TITLE
handle control_group tracking

### PR DESCRIPTION
**Pull request by hackal**

### CODE REVIEW

**Comment by hackal** [comment]
Add control group tracking. Idea is that if you have AB test you just copy paste the banner from variant A and only add `control_group` attribute. Show will be tracked the same way as non control_group.

File: src/exp.js [hunk 1]
```diff
@@ -74,6 +74,8 @@ class Exp {
         this.methods = settings.methods || {};
         /* Formatters */
         this.formatters = settings.formatters || {};
+        /* Control group */
+        this.control_group = settings.control_group || false
```
Comment (lukasotocerny) [inline src/exp.js]: add a semicolon please

Comment (hackal) [reply src/exp.js]: Fixed

**Comment by rkluvanec** [comment]
@lukasotocerny @hackal can we please now merge?

**Pull request merged**